posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,600 @@
+---
+layout: post
+title: "Jakarta EE 10, MicroProfile 6, and Java SE 20 support in Open Liberty 23.0.0.3"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mbroz2
+author_github: https://github.com/mbroz2
+seo-title: Jakarta EE 10, MicroProfile 6, and Java SE 20 support in Open Liberty 23.0.0.3 - OpenLiberty.io
+seo-description: We're thrilled to announce the release of Open Liberty 23.0.0.3, one of the biggest releases of the runtime since its debut over 5 years ago!  This release includes Jakarta EE 10 (Core Profile, Web Profile, and Platform), MicroProfile 6, and Java SE 20 support.
+blog_description: We're thrilled to announce the release of Open Liberty 23.0.0.3, one of the biggest releases of the runtime since its debut over 5 years ago!  This release includes Jakarta EE 10 (Core Profile, Web Profile, and Platform), MicroProfile 6, and Java SE 20 support.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Jakarta EE 10, MicroProfile 6, and Java SE 20 support in Open Liberty 23.0.0.3
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+We're thrilled to announce the release of Open Liberty 23.0.0.3, one of the biggest releases of the runtime since its debut over 5 years ago!  This release includes Jakarta EE 10 Platform, Web Profile, and, for the first time, Core Profile.  Many new enhancements have been added throughout the majority of the features that make up the profiles.
+
+MicroProfile 6, which includes Jakarta EE 10 Core Profile, also makes its formal debut in the 23.0.0.3 release.  It includes the new MicroProfile Telemetry 1.0 feature as well as updates to the Metrics, OpenAPI, and JWT Authentication specifications.

```

Comment:
So have we agreed yet if it is "Jakarta EE 10 Core Profile" or "Jakarta EE Core Profile 10"?! ;-)